### PR TITLE
Add include in nginx.conf

### PR DESCRIPTION
**Pull request by unblee**
Added 'include' in 'nginx.conf' to improve setting flexibility.

### CODE REVIEW

**Comment by dtan4** [comment]
Did you check the behavior?

**Comment by unblee** [comment]
Sorry... I did not check the behavior.
There seems to be a problem. I'll fix.

**Comment by unblee** [comment]
Fixed!
@dtan4 Please review again 🙇

File: Dockerfile [hunk 1]
```diff
@@ -2,7 +2,8 @@ FROM nginx:1.11.9-alpine
 
 # for htpasswd command
 RUN apk add --no-cache --update \
-      apache2-utils
+      apache2-utils \
+    && rm -f /etc/nginx/conf.d/*
```
Comment (dtan4) [inline Dockerfile]: This `rm` command doesn't seems to be related to the above `apk` step. Could you separate it to another `RUN` step?

**Comment by dtan4** [comment]
If Nginx reads `/etc/nginx/conf.d/*` without any special configuration, we can use the current image as it is.
How about that?

**Comment by unblee** [comment]
The build options of nginx in the current image are as follows:

```
$ docker run --rm -it nginx:1.11.9-alpine nginx -V
nginx version: nginx/1.11.9
built by gcc 5.3.0 (Alpine 5.3.0)
built with OpenSSL 1.0.2j  26 Sep 2016
TLS SNI support enabled
configure arguments: --prefix=/etc/nginx --sbin-path=/usr/sbin/nginx --modules-path=/usr/lib/nginx/modules --conf-path=/etc/nginx/nginx.conf --error-log-path=/var/log/nginx/error.log --http-log-path=/var/log/nginx/access.log --pid-path=/var/run/nginx.pid --lock-path=/var/run/nginx.lock --http-client-body-temp-path=/var/cache/nginx/client_temp --http-proxy-temp-path=/var/cache/nginx/proxy_temp --http-fastcgi-temp-path=/var/cache/nginx/fastcgi_temp --http-uwsgi-temp-path=/var/cache/nginx/uwsgi_temp --http-scgi-temp-path=/var/cache/nginx/scgi_temp --user=nginx --group=nginx --with-http_ssl_module --with-http_realip_module --with-http_addition_module --with-http_sub_module --with-http_dav_module --with-http_flv_module --with-http_mp4_module --with-http_gunzip_module --with-http_gzip_static_module --with-http_random_index_module --with-http_secure_link_module --with-http_stub_status_module --with-http_auth_request_module --with-http_xslt_module=dynamic --with-http_image_filter_module=dynamic --with-http_geoip_module=dynamic --with-http_perl_module=dynamic --with-threads --with-stream --with-stream_ssl_module --with-stream_ssl_preread_module --with-stream_realip_module --with-stream_geoip_module=dynamic --with-http_slice_module --with-mail --with-mail_ssl_module --with-compat --with-file-aio --with-http_v2_module
```
IMO, Since `--conf-path=/etc/nginx/nginx.conf` is specified in the build option, `/etc/nginx/conf.d/ * ` will never be read as default.

File: Dockerfile (hunk 1 above)
Comment (unblee) [reply Dockerfile]: Fixed.

**Comment by dtan4** [comment]
> /etc/nginx/conf.d/ * will never be read as default.

I see.

LGTM 👍 Thanks ❗️

**Pull request merged**